template/base/next-auth.d.ts

```diff
@@ -0,0 +1,12 @@
+import { DefaultSession } from "next-auth";
```

Comment:
`next-auth.d.ts` should probably not be included in `base/` since it should not be always present. 
I was not sure to how to add it conditionally. Please give me some idea as to how I can do it, I will update the PR then.

Reply:
I think these would be the steps in order to do it conditionally:
1. move the file to `template/addons/next-auth` folder
2. in `src/installers/next-auth.ts`, copy the file by using the same pattern as done with the others files:
   - add src and dest variables (be sure to set the correct paths)
   - copy the file by adding the call in the `Promise.all` array

This way the file should only be copied when the next-auth package in installed.

Reply:
Thanks, @ochicf, it helped a lot! 
I have done the changes